src/App.jsx

```diff
@@ -1,68 +1,70 @@
-import React from "react";
-import { motion } from "framer-motion";
-
-import styles from "./style";
-import {
-  Navbar,
-  Hero,
-  Education,
-  SkillsAndExperience,
-  ExtraCurricular,
-  Footer,
-  OpenSource,
-  Projects,
-  Loading
-} from "./components";
-
-const App = () => {
-  const [isLoading,setIsLoading] = React.useState(true);
-
-  React.useEffect(()=>{
-    setTimeout(()=>{setIsLoading(false)},1600);
-  },[])
-
-  if(isLoading){
-    return (
-      <div className="bg-primary w-full overflow-hidden">
-        <Loading/>
-      </div>
-    ) 
-  }else{
-    return (
-      // A div to wrap the entire application
-    <div className="bg-primary w-full overflow-hidden">
-      <motion.section
-        initial={{ x: -100, opacity: 0.25 }}
-        animate={{ x: 0, opacity: 1 }}
-        transition={{ duration: 1 }}
-      >
-        <div className={`${styles.paddingX} ${styles.flexCenter}`}>
-          <div className={`${styles.boxWidth}`}>
-            <Navbar />
-          </div>
-        </div>
-
-        <div className={`bg-primary ${styles.flexStart}`}>
-          <div className={`${styles.boxWidth}`}>
-            <Hero />
-          </div>
-        </div>
-
-        <div className={`bg-primary ${styles.flexCenter} ${styles.paddingX} `}>
-          <div className={`${styles.boxWidth}`}>
-            <SkillsAndExperience />
-            <Education />
-            <Projects />
-            <OpenSource />
-            <ExtraCurricular />
-          </div>
-        </div>
-        <Footer />
-      </motion.section>
-    </div>
-
-    );
-  }
-};
-
-export default App;
+import React from "react";
+import { motion } from "framer-motion";
+
+import styles from "./style";
+import {
+  Navbar,
+  Hero,
+  Education,
+  SkillsAndExperience,
+  ExtraCurricular,
+  Footer,
+  OpenSource,
+  Projects,
+  BlogPosts,
+  Loading
+} from "./components";
+
+const App = () => {
+  const [isLoading,setIsLoading] = React.useState(true);
+
+  React.useEffect(()=>{

```

Comment:
Are there so many changes to this file because of indentation?
Same with constants/index.js

Reply:
It was the wrong EOL format being used. Fixed in [a424987](https://github.com/mittal-parth/personal-portfolio/pull/40/commits/a4249877399aa44b093c7d492148d80a78101b31)